src/app/components/OptimoPromos/Image/index.styles.jsx

```diff
@@ -0,0 +1,8 @@
+import styled from '@emotion/styled';
+import Image from '#legacy/psammead-image/src';
+
+export default styled(Image)`
+  aspect-ratio: 16 / 9;
+  object-fit: cover;
+  height: auto;
```

Comment:
I would have thought these styles are already part of the Image component

Reply:
The `aspect-ratio: 16/9` is definitely not in the Image component and this is causing a bug described in the README.md.
The height auto is needed, otherwise, the images will not resize correctly.

object-fit I believe can be removed!!